kernel/timeslicing.c

```diff
@@ -58,11 +58,12 @@ static void slice_timeout(struct _timeout *timeout)
 	slice_expired[cpu] = true;
 
 	/* We need an IPI if we just handled a timeslice expiration
-	 * for a different CPU.  Ideally this would be able to target
-	 * the specific core, but that's not part of the API yet.
+	 * for a different CPU.  Ideally the underlying architecture
+	 * allows for the IPI needed for CPU <cpu> to be directed
+	 * only to it and not any other CPUs.
```

Comment:
Why "ideally"?  That's the way it works now.  My comment was probably silly to begin with, but the limitation it was trying to document doesn't exist anymore.  Just remove.  "We need an IPI if we just handled a timeslice expiration for a different CPU" is all that needs be said.